pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/models/MarcRecord.scala

```diff
@@ -6,7 +6,15 @@ package weco.pipeline.transformer.marc_common.models
  * */
 trait MarcRecord {
 
+  val leader: String
```

Comment:
Adding the leader and control fields better matches the requirements of a MARC Record, but is missing from Sierra data.